invenio/templates/application.yaml

```diff
@@ -102,6 +112,8 @@ spec:
         volumeMounts:
           - name: uwsgi-config
             mountPath: '/opt/invenio/src/uwsgi'
+          - name: "invenio-assets"
```

Comment:
@ppanero Sorry for the noob question: The web pod needs to mount the "invenio-assets" volume in the web pod for what? I understand that you need to provide the volume so the `initContainer` can copy files in it right? But the web container needs to mount it too for which reason?

Also I assume that as long as  the `invenio-assets` volume is actually the "invenio-assets-claim" persistent volume, the copy that is happening in the `initContainer` will persist the data there?
The persistent volume is created/mounted only once or for every web pod and in the end it persists the data the last pod deposited into it?

Reply:
Discussed IRL

Reply:
Discussion point: Yes, Zach is right. However, since Nginx might become a sidecar (CC @Glignos) there is no point on changing this now, let's see the outcome of the sidecar work :)